docs/content/latest/releases/whats-new/latest-release.md

```diff
@@ -0,0 +1,132 @@
+
+### Known Issues
+
+#### Yugabyte Platform
+
+N/A
+
+#### Core Database
+
+N/A
+
+## Notes
+
+{{< note title="New release versioning" >}}
+
+Starting with v2.2, Yugabyte release versions follow a [new release versioning convention](../../versioning). The latest release series, denoted by `MAJOR.ODD`, incrementally introduces new features and changes and is intended for development and testing only. Revision releases, denoted by `MAJOR.ODD.REVISION` versioning, can include new features and changes that might break backwards compatibility. For more information, see [Supported and planned releases](../../releases-overview).
+
+{{< /note >}}
+
+## Not yet categorized
+
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] Remove TTL-related computations from the PgSQL read path
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] Remove table_tombstone_time_ from DocRowwiseIterator and delegate state to DocDBTableReader
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] docdb - Pull DocDBTableReader into DocRowwiseIterator
+* [[4944](https://github.com/yugabyte/yugabyte-db/issues/4944)] docdb - Refactor table tombstone and ancestor scanning logic from doc_reader.cc
+* [[5383](https://github.com/yugabyte/yugabyte-db/issues/5383)] YSQL: Jepsen-like long fork test
+* [[5535](https://github.com/yugabyte/yugabyte-db/issues/5535)] [Platform] Remove stale instance configs after cloud provider is deleted (#6975)
+* [[5624](https://github.com/yugabyte/yugabyte-db/issues/5624)] Show pricing info for Azure
+* [[5626](https://github.com/yugabyte/yugabyte-db/issues/5626)] Enable Hosted Zone for Azure
+* [[5838](https://github.com/yugabyte/yugabyte-db/issues/5838)] [Platform] Add node link to corresponding Azure portal URL
+* [[5840](https://github.com/yugabyte/yugabyte-db/issues/5840)] Add user tags for Azure universes
+* [[5841](https://github.com/yugabyte/yugabyte-db/issues/5841)] Show only Azure VMs that meet minimum requirements
+* [[5922](https://github.com/yugabyte/yugabyte-db/issues/5922)] YCQL: Improve audit logging
+* [[6321](https://github.com/yugabyte/yugabyte-db/issues/6321)] [Platform] Support using Shared Gallery Images when creating Azure universes
+* [[6681](https://github.com/yugabyte/yugabyte-db/issues/6681)] [Platform] Show kubectl command for Kubernetes pods in Connect modal (#7506)
+* [[6712](https://github.com/yugabyte/yugabyte-db/issues/6712)] Fix issue with JDK incompatibility in Java RPC client
+* [[6756](https://github.com/yugabyte/yugabyte-db/issues/6756)] [Platform] Created date showing up as "Invalid Date". (#7158)
+* [[7024](https://github.com/yugabyte/yugabyte-db/issues/7024)] [Platform] Unable to edit number of nodes in AZ section (#7350)
+* [[7054](https://github.com/yugabyte/yugabyte-db/issues/7054)] [YW] Add conditional checks for hiding specific platform elements in non-platform mode.
+* [[7126](https://github.com/yugabyte/yugabyte-db/issues/7126)] PITR: Create snapshot for schedule
```

Comment:
db feature work -- WIP on the PITR feature, not sure if we want to squash these somehow, as there's going to be a couple of them

eg: one way I can think of is to have a PITR section under new features and just list the progress we've made since the last time, as far as new functionality

Reply:
made a bullet in new features, added sub-bullets to that for 7121 and 7125